src/controllers/EntryController.ts

```diff
@@ -0,0 +1,12 @@
+import { Request, Response, NextFunction } from 'express';
+import { controller, httpGet, BaseHttpController } from 'inversify-express-utils';
+import { PENALTY_DETAILS_PREFIX, ENTRY_PREFIX } from '../utils/Paths';
+
+@controller(ENTRY_PREFIX)
+export class EntryController extends BaseHttpController{
+    
+    @httpGet('')
+    public renderView(req: Request, res: Response, next: NextFunction): void {
```

Comment:
You can remove the unused parameters

Reply:
Name is not quire right as well, it does not render anything